src/Symfony/Component/Routing/Generator/UrlGenerator.php

```diff
@@ -37,6 +38,8 @@ class UrlGenerator implements UrlGeneratorInterface, ConfigurableRequirementsInt
 
     protected $logger;
 
+    protected $defaultLocale;
```

Comment:
should be private I suppose (like we do for all new properties, no matter the previous ones)